toxcore/tox.h

```diff
@@ -689,6 +689,19 @@ struct Tox_Options {
      * Default: false.
      */
     bool experimental_groups_persistence;
+
+    /**
+     * @brief Allow passing hostnames to the bootstrap function and proxy host option.
```

Comment:
Doesn't mention that it applies to `tox_add_tcp_relay()` too, unless it got grouped together with bootstrap?

Reply:
Done.